src/libs/elektra/keyname.c

```diff
@@ -1255,23 +1309,32 @@ ssize_t keyGetBaseNameSize (const Key * key)
 
 
 /**
- * Calculate the basename of a key name and put it in @p returned finalizing
- * the string with NULL.
+ * Calculate the basename of a Key's name and copy it to @p returned
```

Comment:
I think this is much simpler:
```suggestion
 * Copy the Key's basename to @p returned
```
Any details could be in the long description below.

Reply:
fixed in https://github.com/ElektraInitiative/libelektra/pull/3795/commits/cea2c469decb39e99b3d71197f27ed40dede38d9